xpdacq/analysis.py

```diff
@@ -141,7 +138,7 @@ def save_tiff(headers, dark_subtraction = True, *, max_count = None):
             else:
                 print('Sorry, something went wrong with your tif saving')
                 return
-            if max_count is not None and ind<= max_count:
```

Comment:
@chiaohaoliu if you are finished with commits/discussion and want something
pulled in, please could you make that request?  I would like to be pulling
things in in a more granular fashion than we have....for example, this
small change to ``save_tiff``, and it is easier to do that if I know that
it is passing all tests, functional and unit, and is ready to be pulled in.

S

On Mon, Mar 14, 2016 at 1:48 PM, Thomas A Caswell <notifications@github.com>
wrote:

> In xpdacq/analysis.py
> <https://github.com/xpdAcq/xpdAcq/pull/136#discussion_r56043933>:
>
> > @@ -141,7 +138,7 @@ def save_tiff(headers, dark_subtraction = True, *, max_count = None):
> >              else:
> >                  print('Sorry, something went wrong with your tif saving')
> >                  return
> > -            if max_count is not None and ind<= max_count:
>
> See, never trust me ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xpdAcq/xpdAcq/pull/136/files#r56043933>.
>



-- 
----
Prof. Simon Billinge

Applied Physics & Applied Mathematics
Columbia University
500 West 120th Street
Room 200 Mudd, MC 4701
New York, NY 10027
Tel: (212)-854-2918 (o) 851-7428 (lab)

Condensed Matter Physics and Materials Science Dept.
Brookhaven National Laboratory
P.O. Box 5000
Upton, NY 11973-5000
(631)-344-5661

email: sb2896 at columbia dot edu
home: http://thebillingegroup.com